### PR TITLE
docs: improve some mqtt options description

### PR DESCRIPTION
**Pull request by HJianBo**
Fixes https://emqx.atlassian.net/browse/EMQX-9322

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- ~Added tests for the changes~
- ~Changed lines covered in coverage report~
- ~Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` and `.zh.md` files~
- [x] For internal contributor: there is a jira ticket to track this change
- ~If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up~
- ~Schema changes are backward compatible~

## Checklist for CI (.github/workflows) changes

- ~If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)~
- ~Change log has been added to `changes/` dir for user-facing artifacts update~

### CODE REVIEW

**Comment by HJianBo** [comment]
Could you help to review these changes @LenaLenaPan @zmstone

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 1]
```diff
@@ -783,8 +783,8 @@ but do not send any data."""
 
     mqtt_ignore_loop_deliver {
         desc {
-            en: """Ignore loop delivery of messages for MQTT v3.1.1/v3.1.0, similar to <code>No Local</code> subscription option in MQTT 5.0."""
-            zh: """是否为 MQTT v3.1.1/v3.1.0 客户端忽略投递自己发布的消息，类似于 MQTT 5.0 中的 <code>No Local</code> 订阅选项。"""
+            en: """Whether the messages sent by the MQTT v3.1.1/v3.1.0 client will be forwarded to the client itself, similar to No Local in MQTT 5.0."""
```
Comment (LenaLenaPan) [inline apps/emqx/i18n/emqx_schema_i18n.conf:786]: shall we use the code format for no local?

Comment (LenaLenaPan) [reply apps/emqx/i18n/emqx_schema_i18n.conf:786]: also applies to the Chinese version

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 2]
```diff
@@ -807,8 +807,10 @@ When set to true, invalid utf8 strings in for example client ID, topic name, etc
 
     mqtt_response_information {
         desc {
-            en: """Specify the response information returned to the client. This feature is disabled if is set to \"\". Applies only to clients using MQTT 5.0."""
-            zh: """指定返回给客户端的响应信息。如果设置为 \"\"，则禁用此功能。仅适用于使用 MQTT 5.0 协议的客户端。"""
+            en: """UTF-8 string, for creating the response topic, for example, if set to <code>reqrsp/</code>,  the publisher/subscriber will communite under the topic <code>reqrsp/</code>.
```
Comment (LenaLenaPan) [inline apps/emqx/i18n/emqx_schema_i18n.conf:810]:
```suggestion
            en: """UTF-8 string, for creating the response topic, for example, if set to <code>reqrsp/</code>,  the publisher/subscriber will communite under the topic <code>reqrsp/</code>. Only applicable to MQTT 5.0 clients. 
```

File: apps/emqx/i18n/emqx_schema_i18n.conf (hunk 1 above)
Comment (zmstone) [inline apps/emqx/i18n/emqx_schema_i18n.conf:786]:
```suggestion
            en: """Whether the messages sent by the MQTT v3.1.1/v3.1.0 client will be looped back to the publisher itself, similar to No Local in MQTT 5.0."""
```

File: apps/emqx/i18n/emqx_schema_i18n.conf (hunk 2 above)
Comment (zmstone) [inline apps/emqx/i18n/emqx_schema_i18n.conf:810]:
```suggestion
            en: """UTF-8 string, for creating the response topic, for example, if set to <code>reqrsp/</code>, the publisher/subscriber will communicate using the topic prefix <code>reqrsp/</code>.
```

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 3]
```diff
@@ -783,8 +783,8 @@ but do not send any data."""
 
     mqtt_ignore_loop_deliver {
         desc {
-            en: """Ignore loop delivery of messages for MQTT v3.1.1/v3.1.0, similar to <code>No Local</code> subscription option in MQTT 5.0."""
-            zh: """是否为 MQTT v3.1.1/v3.1.0 客户端忽略投递自己发布的消息，类似于 MQTT 5.0 中的 <code>No Local</code> 订阅选项。"""
+            en: """Whether the messages sent by the MQTT v3.1.1/v3.1.0 client will be forwarded to the client itself, similar to No Local in MQTT 5.0."""
+            zh: """设置由 MQTT v3.1.1/v3.1.0 客户端发布的消息是否将转发给其本身；类似 MQTT 5.0 协议中的 No Local 选项。"""
```
Comment (HJianBo) [inline apps/emqx/i18n/emqx_schema_i18n.conf:787]:
```suggestion
            zh: """设置由 MQTT v3.1.1/v3.1.0 客户端发布的消息是否将转发给其本身；类似 MQTT 5.0 协议中的 <code>No Local</code> 选项。"""
```

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 4]
```diff
@@ -807,8 +807,10 @@ When set to true, invalid utf8 strings in for example client ID, topic name, etc
 
     mqtt_response_information {
         desc {
-            en: """Specify the response information returned to the client. This feature is disabled if is set to \"\". Applies only to clients using MQTT 5.0."""
-            zh: """指定返回给客户端的响应信息。如果设置为 \"\"，则禁用此功能。仅适用于使用 MQTT 5.0 协议的客户端。"""
+            en: """UTF-8 string, for creating the response topic, for example, if set to <code>reqrsp/</code>, the publisher/subscriber will communicate using the topic prefix <code>reqrsp/</code>.
+To disable this feature, input <code>\"\"</code> in the text box below."""
```
Comment (HJianBo) [inline apps/emqx/i18n/emqx_schema_i18n.conf:811]:
```suggestion
To disable this feature, input <code>\"\"</code> in the text box below. Only applicable to MQTT 5.0 clients."""
```

**Review by LenaLenaPan — APPROVED** [review]

**Review by zmstone — APPROVED** [review]

**Review by zmstone — APPROVED** [review]

**Pull request merged**